core-tests/shared/src/test/scala/zio/ZIOBaseSpec.scala

```diff
@@ -1,8 +1,45 @@
 package zio
 
+import scala.annotation.tailrec
+
 import zio.duration._
 import zio.test._
 
 trait ZIOBaseSpec extends DefaultRunnableSpec {
   override def aspects = List(TestAspect.timeout(60.seconds))
+
+  sealed trait ZIOTag {
+    val value: String
+    val subTags: List[ZIOTag] = Nil
+  }
+  object ZIOTag {
+    case object Errors        extends ZIOTag { override val value = "errors" }
+    case object FutureInterop extends ZIOTag { override val value = "future" }
```

Comment:
I had to use two words for this tag: `FutureInterop`, I couldn't make it just `Future` because it would clash with scala `Future`, do you have any ideas for making this just one word @adamgfraser @jdegoes?